js/constants/AppConstants.js

```diff
@@ -23,3 +23,6 @@ export const RECALL_STAGES = {
   FIRST: 'recallFirstText',
   NONE : 'recallNoneText'
 };
+
+export const COOKIE_LAST_STATE = 'last_state';
```

Comment:
While this may be truly preference on my part as cookies are set to the domain (therefore making it not really matter), do we want to prefix this in any way to identify it further?